_sections/act_and_prepare/en/02-household-practices/02-keep-your-home-clean.md

```diff
@@ -1,14 +1,3 @@
 ### Keep your home clean and disinfect
 
-[You can find CDC Recommendations
-here.](https://www.cdc.gov/coronavirus/2019-ncov/prepare/cleaning-disinfection.html)
-
-Clean AND disinfect frequently touched surfaces daily. This includes tables, doorknobs, light switches, countertops, handles, desks, phones, keyboards, toilets, faucets, and sinks.
-If surfaces are dirty, clean them with detergent or soap and water prior to disinfection.
-
-
-To make a bleach solution, mix 5 tablespoons (1/3 cup) bleach per gallon of water OR 4 teaspoons bleach per quart of
-water.
-
-If you have the ability to do so, consider a 'staging' area for clean entry into your home. Remove shoes, outerwear. If you work in a high risk environment like a hospital, nursing home, etc, wash your clothes immediately and then dry them at high heat or at least dry in the sunshine. Wash hands right away. Isolate your mail, packages and purchases (including food) before bringing them into the house and leave them isolated a few hours at a minimum. If you are in an especially high risk group, you might want to consider leaving goods isolated a longer time (a few days) and avoid eating raw produce, and also decontaminating products that come into your house.
-The latest [research](https://www.medrxiv.org/content/10.1101/2020.03.09.20033217v1.full.pdf) shows that the virus can survive up to 4 hours on copper surfaces, up to 3 days on plastic and stainless steel, and up to 24 hours on cardboard.
+Surface contamination while possible has not been shown to be the main mechanism of infection. That said, in laboratory environments, [research](https://www.medrxiv.org/content/10.1101/2020.03.09.20033217v1.full.pdf) the virus has been shown to survive on surfaces. Therefore if you're at high risk or work in a high-risk environment, is still a good idea to have a 'staging' area for clean entry into your home. Remove shoes, outerwear. Isolate your mail, packages and purchases at least a few hours, especially if you don't need them right away. 
```

Comment:
_That said, in laboratory environments, research the virus has been shown to survive on surfaces._

The above sentences doesn't parse well.


Do we want to keep links for resources on how to clean/disinfect common surfaces etc or do we think this is not necessary?

Reply:
I don't think that it is necessary

Reply:
In a typical home environment, that is.